### PR TITLE
refactor(cli/repl): get context id from notification

### PR DESCRIPTION
**Pull request by caspervonb**
This takes the execution context id from a notification which is sent on Runtime.enable rather than hard-coding it to a magic value.

### CODE REVIEW

File: cli/repl.rs [hunk 1]
```diff
@@ -294,6 +308,10 @@ pub async fn run(
           ),
         }
 
+        for notification in session.notifications() {
+          println!("{}", notification);
```
Comment (bartlomieju) [inline cli/repl.rs:312]: Is this for debug only?

File: cli/repl.rs [hunk 2]
```diff
@@ -89,8 +89,7 @@ pub async fn run(
   global_state: &GlobalState,
   mut worker: MainWorker,
 ) -> Result<(), AnyError> {
-  // Our inspector is unable to default to the default context id so we have to specify it here.
-  let context_id: u32 = 1;
+  let mut context_id: u64 = 0;
```
Comment (bartlomieju) [inline cli/repl.rs:92]: Is it safe to have `0` here if `executionContextCreated` is not received?

File: cli/repl.rs (hunk 1 above)
Comment (caspervonb) [reply cli/repl.rs:312]: Oops, indeed.

File: cli/repl.rs (hunk 2 above)
Comment (caspervonb) [reply cli/repl.rs:92]: Implementation defined but for us 0 is an invalid context identifier, we start at 1 so this is the uninitialized value.

Per spec, executionContextCreated must always be sent for each context after Runtime.enable has been called.

Comment (bartlomieju) [reply cli/repl.rs:92]: Cool, can you add that comment to code?

**Comment by bartlomieju** [comment]
@caspervonb please rebase

**Review by bartlomieju — APPROVED** [review]
LGTM

**Pull request merged**